test/unit/utils.spec.js

```diff
@@ -496,11 +496,11 @@ describe('lib/utils', function() {
     if (typeof global.Symbol === 'function') {
       it('should handle Symbol', function() {
         var symbol = Symbol('value');
-        expect(stringify(symbol), 'to be', 'Symbol(value)');
+        expect(stringify(symbol), 'to match', /^Symbol\(value\)/);
```

Comment:
`core-js` adds weird characters to the `Symbol` string representation in IE11.  🤷‍♂️